MPF.Core/Utilities/Tools.cs

```diff
@@ -389,7 +389,7 @@ public static bool ParseGetKeyLog(string? logPath, out byte[]? key, out byte[]?
                     return false;
 
                 // Convert PIC to byte array
-                pic = Tools.HexStringToByteArray(discPICStr.Substring(0, 230));
+                pic = discPICStr + "00000000";
```

Comment:
Good question, I should be doing this differently.
3k3y-style PIC, as stored in IRDs, is 115 bytes (230 hex chars) long.
Redump stores 132 bytes (264 chars).
I *was* only reading the first 8 lines (256 chars) and therefore needed to add 8 more zeroes rather than trim it to 230 chars.
However, I should just be reading the 9th line instead.

Reply:
That sounds preferable since it can always be reduced later.